src/Utilities/Registry.js

```diff
@@ -0,0 +1,21 @@
+import ReducerRegistry from './ReducerRegistry';
+let registry;
+
+export default function(settings = {}) {
+  function init(initialState = {}, middleware = []) {
+    if(!registry) {
+      registry = new ReducerRegistry(initialState, [...middleware]);
+    }
+    return registry;
+  }
+  return function(target) {
+    target.prototype.registry = init(settings.initialState, settings.middleware);
+  }
+}
+
+export function getStoreFromRegistry(initialState = {}, middleware = []) {
+  if(!registry) {
+    registry = new ReducerRegistry(initialState, [...middleware]);
+  }
+  return registry.getStore();
```

Comment:
Thanks, @karelhala, this simplifies the setup for Insights UI newcommers!

It feels ugly to me that the line 6-9 are the same as 17-20 except for the getStore() call. Should be no problem extraction is so that the currend `init` can be used in `getStoreFromRegistry`.